install/kubernetes/cilium/templates/cilium-operator-deployment.yaml

```diff
@@ -41,16 +41,10 @@ spec:
 {{- if or (ge .Capabilities.KubeVersion.Minor "14") (gt .Capabilities.KubeVersion.Major "1") }}
       # In HA mode, cilium-operator pods must not be scheduled on the same
       # node as they will clash with each other.
+  {{- if .Values.affinityOperator }}
       affinity:
-        podAntiAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-          - labelSelector:
-              matchExpressions:
-              - key: io.cilium/app
-                operator: In
-                values:
-                - operator
-            topologyKey: "kubernetes.io/hostname"
+  {{ toYaml .Values.affinityOperator | indent 6 }}
```

Comment:
I think this should be `indent 8`

Reply:
@aanm I changed some options, following @sayboras advices. Also `nindent` is now used as the `indent` was breaking some indentation resulting in the test error.